pkgs/tools/misc/fls/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+}:
+
+with import <nixpkgs>
```

Comment:
This isn't allowed in Nixpkgs. You can only use Rust versions already exists in Nixpkgs.